lib/utils/array.js

```diff
@@ -1,3 +1,3 @@
-export default function last(arr) {
+export function last(arr) {
```

Comment:
Hmm, wasn't this used anywhere? I mean changing from default export to named export 🤔